umade-client/src/pages/Authentication/OTPForm.jsx

```diff
@@ -0,0 +1,123 @@
+import {
+	Avatar,
+	Button,
+	CssBaseline,
+	TextField,
+	Grid,
+	Box,
+	Typography,
+	Container,
+	MenuItem,
+	FormControl,
+	useTheme,
+} from '@mui/material'
+import LockOutlinedIcon from '@mui/icons-material/LockOutlined'
+import { useState } from 'react'
+import { useNavigate, Link } from 'react-router-dom'
+import { toast } from 'react-toastify'
+import axios from 'axios'
+import '@fontsource/roboto/300.css'
+import '@fontsource/roboto/400.css'
+import '@fontsource/roboto/500.css'
+import '@fontsource/roboto/700.css'
+import { orange } from '@mui/material/colors'
+import Paper from '@mui/material/Paper'
+import Stack from '@mui/material/Stack'
+import SendIcon from '@mui/icons-material/Send'
+function Copyright(props) {
+	return (
+		<Typography
+			variant="body2"
+			color="text.secondary"
+			align="center"
+			{...props}
+		>
+			{'Copyright © by Umade ' + new Date().getFullYear() + '.'}
+		</Typography>
+	)
+}
+
+function OTPForm() {
+	const [OTP, setOTP] = useState('')
+	const navigate = useNavigate()
+	const paperStyle = {
+		padding: 20,
+		height: '450px',
+		width: 500,
+		margin: '50px auto',
+	}
+	const handleChange = (event) => {
+		setOTP(event.target.value)
+	}
+	const handleOTP = async(event) => {
+		event.preventDefault()
+		const res = await axios.post('api/validateOTP', {
+			otp: OTP,
+		})
```

Comment:
Don't need to put data into body like this, we have configured the content type of request at line 10 index.tsx (axios will automatically convert the input data to correct content type)
```suggestion
		const res = await axios.post('api/validateOTP', OTP)
```